apps/editor/src/app/base.tsx

```diff
@@ -41,6 +42,7 @@ import {
 
 import {PermissionControl} from './atoms/permissionControl'
 import Version from './atoms/version'
+import {fr, de, enUS} from 'date-fns/locale'
```

Comment:
Why are these called locales :( Only enUS is a locale here.

Reply:
Currently using enUS for the translations so it makes sense to match that, no?

Reply:
Oh we do? Then nvm, thats another topic to switch to enGB